easybuild/easyconfigs/r/RepeatMasker/RepeatMasker-4-0-6-foss-2016a-Perl-5.22.1.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PackedBinary'
+
+name = 'RepeatMasker'
+version = '4-0-6'
```

Comment:
@RvDijk this should be `4.0.6`? Please change the easyconfig filename accordingly